nodes/src/nodes/llm_ibm_watson/test_location_injection.py

```
@@ -0,0 +1,122 @@
+    def test_unknown_region_rejected(self, loc):
+        with pytest.raises(ValueError, match='Unknown IBM Cloud location'):
+            _validate_location(loc)
+
+
+# ---- Tests for empty / placeholder values --------------------------------
+
+class TestEmptyAndPlaceholder:
+    """Empty strings and the UI placeholder must be rejected."""
+
+    def test_empty_string(self):
+        with pytest.raises(ValueError):
```

Ruff (0.15.6)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)